backend/Bindings/Edna.Bindings.User/UsersClient.cs

```diff
@@ -0,0 +1,36 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Edna.Utils.Http;
+using Newtonsoft.Json;
+
+namespace Edna.Bindings.User
+{
+    public class UsersClient
+    {
+        private readonly string _baseServiceUriString;
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        internal UsersClient(string serviceUrlEnvironmentVariable, IHttpClientFactory httpClientFactory)
+        {
+            _httpClientFactory = httpClientFactory;
+            _baseServiceUriString = Environment.GetEnvironmentVariable(serviceUrlEnvironmentVariable);
+            if (string.IsNullOrEmpty(_baseServiceUriString))
```

Comment:
Should we also consider checking whether `_baseServiceUriString` is actually a uri string and not a simple string say "Learn-LTI"?